assignments/life_expectancy/main.py

```diff
@@ -47,7 +47,8 @@ def main(i_path, region, o_path): # pragma: no cover
     parser = argparse.ArgumentParser()
     parser.add_argument('-i', help= "Input file path.") 
     parser.add_argument('-o', help = "Output file path.")
-    parser.add_argument("-r", default="PT", help = "Desire region to filter the csv.")
+    parser.add_argument("-r",  type = Region, choices = list(Region), \
```

Comment:
```suggestion
    parser.add_argument("-r",  type = Region, choices = Region, \
```

You don't really need the `list` here

I don't you need the list here

Reply:
Changed